src/main/kotlin/com/terraformation/backend/support/api/SupportController.kt

```diff
@@ -32,9 +43,41 @@ class SupportController(private val service: SupportService) {
       @RequestBody payload: SubmitSupportRequestPayload
   ): SubmitSupportRequestResponsePayload {
     val issueKey =
-        service.submitServiceRequest(payload.description, payload.summary, payload.requestTypeId)
+        service.submitServiceRequest(
+            requestTypeId = payload.requestTypeId,
+            summary = payload.summary,
+            description = payload.description,
+            attachmentIds = payload.attachmentIds ?: emptyList(),
```

Comment:
It makes me nervous to pass an arbitrary client-supplied identifier through to an external service with no validation or permission checking (nothing to stop people from attaching other people's files to their issues!) but I suppose it's probably what the existing support form already does anyway, so this isn't making anything worse. Plus I guess the worst possible outcome of the client forging an ID here is that a Terraformation employee sees the wrong attachment.